src/Services/FHIR/FhirAllergyIntoleranceService.php

```diff
@@ -52,6 +56,58 @@ protected function loadSearchParameters()
         ];
     }
 
+
+    /**
+     * Parses an OpenEMR allergyIntolerance record, returning the equivalent FHIR AllergyIntolerance Resource
+     *
+     * @param array $dataRecord The source OpenEMR data record
+     * @param boolean $encode Indicates if the returned resource is encoded into a string. Defaults to false.
+     * @return FHIRAllergyIntolerance
+     */
+    public function createProvenanceResource($dataRecord = array(), $encode = false)
+    {
+        
+        $allergyProvenance = new FHIRProvenance();
+        $meta = array('versionId' => '1', 'lastUpdated' => gmdate('c'));
+        $allergyProvenance->setMeta($meta);
+        
+        $id = new FHIRId();
+        $uuid = new UuidRegistry();
+        $uuidString = UuidRegistry::uuidToString($uuid->createUuid());
```

Comment:
Does this uuid need to be persistent or is it just a one time thing?

Reply:
I do not think it needs to be a persistent as this resource is always dependent on the target resource. Just my assumption.

Reply:
Looked this up and I think I agree with your assumption :)
Since this is basically a random uuid that will not be persistent, then would have it skip logging/tracking, so would change above 2 lines to:
```
$uuidString = UuidRegistry::uuidToString((new UuidRegistry(['disable_tracker' => true]))->createUuid());
```

Reply:
Made the suggested changes. For some reason, the Travis checks have been running for a long time now